product_warranty/i18n/es_VE.po

```diff
@@ -0,0 +1,16 @@
+# Translation of Odoo Server.
```

Comment:
To avoid the warning "es_**" do not have translations, on this case there is not any different terms, but without the file the warning appears always. We use it es_MX and es_PA and es_VE